src/components/atoms/button.tsx

```diff
@@ -1,7 +1,30 @@
-import React from 'react'
+import React, { Children, ReactNode } from "react";
+import { ButtonStyles } from "../../common/styles";
```

Comment:
So we like to separate our external imports from our internal ones.

So we just keep 1 space in between.
External being at the top.
Space
Then internal imports